core/modules/update/update.admin.inc

```diff
@@ -13,50 +13,93 @@
+      '0' => t('Do not check automatically.'),
     ),
-    '#description' => t('Select how frequently you want to automatically check for new releases of your currently installed modules, themes, and layouts.'),
   );
-
-  $form['update_check_disabled'] = array(
+  $form['check']['update_check_disabled'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Check for updates of disabled and uninstalled modules, themes, and layouts.'),
+    '#title' => t('Also check for updates of disabled modules, themes, and layouts.'),
     '#default_value' => $config->get('update_disabled_extensions'),
   );
 
+  $form['email'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Update Notifications'),
+    '#description' => t('This website can send notifications via e-mail when updates are available.'),
+    '#states' => array(
+      'visible' => array(
+        'input[name=update_check_frequency]' => array('!value' => 0),
+      ),
+    ),
+  );
+  $form['email']['update_notification_threshold'] = array(
+    '#type' => 'radios',
+    '#title' => t('When should this website send e-mail notifications?'),
+    '#default_value' => $config->get('update_threshold'),
+    '#options' => array(
+      'all' => t('When any update is available.'),
+      'security' => t('Only if a security update is available.'),
+      'none' => t('Do not send email notifications.'),
+    ),
+  );
   $notify_emails = $config->get('update_emails');
   if (empty($notify_emails)) {
     $notify_emails = array();
   }
-  $form['update_notify_emails'] = array(
+  $form['email']['update_notify_emails'] = array(
     '#type' => 'textarea',
     '#title' => t('E-mail addresses to notify when updates are available'),
     '#rows' => 4,
     '#default_value' => implode("\n", $notify_emails),
-    '#description' => t('Whenever your site checks for available updates and finds new releases, it can notify a list of users via e-mail. Put each address on a separate line. If blank, no e-mails will be sent.'),
+    '#description' => t('Put each email address on a separate line. If blank, no e-mails will be sent.'),
+    '#states' => array(
+      'visible' => array(
+        'input[name=update_notification_threshold]' => array('!value' => 'none'),
+      ),
+    ),
   );
 
-  $form['update_notification_threshold'] = array(
-    '#type' => 'radios',
-    '#title' => t('E-mail notification threshold'),
-    '#default_value' => $config->get('update_threshold'),
-    '#options' => array(
-      'all' => t('All newer versions'),
-      'security' => t('Only security updates'),
+  $docs = l('File permissions and ownership', 'https://backdropcms.org/file-permissions-and-ownership');
+  $form['self_update'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Self-updates'),
+    '#description' => t('This website has the ability to update itself, but the file system permissions must be set properly. See: !link for more details.',
+      array('!link' => $docs)),
```

Comment:
...I don't think that putting this in a separate line is according to coding standards.

Reply:
If the line exceeds 80chars it's best to do it this way. Since we can't break the string in the `t()` at a sensible location, breaking after the string means that devs won't need to scroll over to the right to find out what was passed in to `t()` as the second parameter.